tests/testthat/test-report.htest-chi2.R

```diff
@@ -1,98 +1,3 @@
-test_that("report.htest-chi2 report_effectsize", {
```

Comment:
Why were these tests deleted?

Reply:
Initially I only wrote tests for `report_effectsize` since `report` was not working, but now that I got `report` working, `report_effectsize` tests are redundant because they are included in `report`, so this does not reduce coverage since they all tested the same stuff, so I think this is more parsimonious?